.teamcity/settings.kts

```diff
@@ -351,16 +345,6 @@ object RunAllUnitTests : BuildType({
 		}
 		perfmon {
 		}
-		pullRequests {
```

Comment:
This plugin was getting a list of open Pull Requests and creating "fake" branches named `pull/<id>`, so other parts of TeamCity (for example, any `branchFilter`) can filter them.